plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/rating-filter.block_theme.spec.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+
+test.describe( 'Product Filter: Rating Filter Block', async () => {
+	test.describe( 'frontend', () => {
+		test( 'Renders a checkbox list with the available ratings', async ( {
+			page,
+		} ) => {
+			await page.goto( '/product-filters-rating-block/' );
+			const ratingStars = page.locator(
+				'.wc-block-components-product-rating__stars'
+			);
```

Comment:
Can we replace css class locator here (and below) with a role base or label base locator? This particular element has `aria-label` set to `Rated %d out of 5`. By doing so, if we refactor the markup of stars we don't need to update this test.